cmd/stats.go

```diff
@@ -39,6 +40,11 @@ var statsCmd = &cobra.Command{
 	3. Total Dependencies: Total number of dependencies of the mainModule(s)
 	4. Max Depth of Dependencies: Length of the longest chain starting from the first mainModule; defaults to length from the first module encountered in "go mod graph" output`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		//returns error if
+		if len(args) != 0 {
+			return errors.New("Invalid input. This command does not take any argument as an input.")
```

Comment:
I think we should go with `fmt.Errorf` because then we won't need to import another package here. Also please see https://github.com/kubernetes-sigs/depstat/issues/38#issuecomment-902487272